src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/haflows/HaFlowPathsSpec.groovy

```diff
@@ -38,4 +35,30 @@ class HaFlowPathsSpec extends HealthCheckSpecification {
         def actualException = thrown(HttpClientErrorException)
         new HaFlowNotFoundExpectedError(~/HA-flow ${flowId} not found\./).matches(actualException)
     }
+
+    @Tidy
+    def "HA flow main path is not overlapped with protected path"() {
+        assumeTrue(useMultitable, "HA-flow operations require multiTable switch mode")
```

Comment:
Remove this, single-table mode is already obsolete in kilda.

Reply:
Done.
Do we need to remove this verification from the rest of the Ha-flow-related tests?

Reply:
I think for now it's not necessary (unless you want to solve a bunch of conflicts after eventual merge/rebase).